shared/provision/routes.tsx

```diff
@@ -14,7 +14,7 @@ export const newRoutes = {
   error: {getScreen: (): typeof ErrorView => require('./error/container').default},
   forgotUsername: {getScreen: (): typeof ForgotUsername => require('./forgot-username/container').default},
   gpgSign: {getScreen: (): typeof GpgSign => require('./gpg-sign/container').default},
-  paperkey: {getScreen: (): typeof Paperkey => require('./paper-key').default},
+  paperkey: {getScreen: (): typeof Paperkey => require('./paper-key/container').default},
```

Comment:
we should get around to prefixing these with `provision`

Reply:
yeah, it'l lbe better when all the routes are actually typed w/ ownProps, i'll make a ticket to continue that work